apps/api/urls.py

```diff
@@ -11,5 +11,6 @@
 
 urlpatterns = [
     path("participants/<str:participant_id>/", views.update_participant_data, name="update-participant-data"),
+    path("openai/<str:experiment_id>/chat/completions", openai.chat_completions, name="openai-chat-completions"),
```

Comment:
Is the reason for the `openai/` part to indicate that this follows the openai API?

Reply:
It's not really necessary at all, I just thought it was nice to have it in the URL as a clear indication that it's meant to be an OpenAI compatible API.